config/routes.rb

```diff
@@ -6,6 +6,7 @@
 
   get "/get-started" => "pages#get_started"
   get "/features" => "pages#features"
+  get "/forthcoming-features" => "pages#forthcoming-features"
```

Comment:
Annoyingly this needs to be an underscore (even if the URL has a hyphen).

```suggestion
  get "/forthcoming-features" => "pages#forthcoming_features"
```